.env.example

```
@@ -0,0 +1,45 @@
+# Core
+PORT=5050
+NODE_ENV=development
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the PORT key

(UnorderedKey)